### ISSUE TITLE
Downloads not starting

### ISSUE THREAD

**Issue by BigBenlolz** [ModOrganizer2/modorganizer#1478]
So whenever I want to download through MO, it just redirects but nothing happens, download doesnt start. I tried relinking my account, but after one restart it says N/A, like it forgets my account. 

Here the relevant MO log:

**Comment by LostDragonist** [comment]
Please go to Settings > Diagnostics and set the "Log Level" to "Debug."  Click OK to store the setting.  Restart MO.  Try a new download.  Upload the new log.

**Comment by BigBenlolz** [comment]
[mo_interface.log](https://github.com/ModOrganizer2/modorganizer/files/6207273/mo_interface.log)

**Comment by nmeocisco** [comment]
I recently experienced the same when I updated to 2.4.2. I get the exact same logs where it seems to hang on "attempt to verify nexus api key" and I don't see a progress bar indicating a download even though there is a tooltip popup saying "download started". When I downgraded to 2.3.2 the downloads started working again.

**Comment by BigBenlolz** [comment]
So it's been quite a while and I'm on 2.4.4, still the same problem.

**Comment by LostDragonist** [comment]
Yes. We've been unable to solve the problem yet. There are some dev builds of 2.5.0 that use Qt6 (instead of Qt5). They *might* solve the problem but I say that out of nothing but blind hope.

**Comment by fredthe42nd** [comment]
I recently started experiencing this bug as well on both v2.5.0 and v2.5.1rc2 of Mod Organizer. Weirdly enough this problem only exists for me in my Skyrim SE instance  and not my Fallout 4 or Starfield instance.
